Automattic/jetpack
# Add wpcom marketplate upsell to plugins on WoA sites

Fixes https://github.com/Automattic/dotcom-forge/issues/6001

## Proposed changes:

Add a prominent upsell banner from /wp-admin/plugin-install.php to the wpcom plugin marketplace



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Create a woa dev site and take it atomic
2. Use instructions in comment below to apply to the atomic site
3. Go to /wp-admin/plugin-install.php
4. See the banner
5. Click the link in the banner - you should go to the plugin marketplace
